teachers_digital_platform/css/organisms/well.less

```diff
@@ -0,0 +1,36 @@
+// Well styling on Activity search page
+
+.o-well {
+    .tdp-activity-search & {
+        background-color: transparent;
+        max-width: none;
+
+        .h5 {
+            font-size: .875rem;
+        }
+
+        .h6 {
```

Comment:
Since you're overriding two of the key features of .h6, I would suggest giving this a meaningful name (perhaps `.activity_meta-key`?), instead of overriding the size for a third time on this element (`body` > `h5` > CF's `.h6` > your `.h6`).

Reply:
I like this.  Duly modified.